tests/pfcwd_input/pfcwd_test_vectors.py

```diff
@@ -44,11 +44,11 @@
 """
 
 pfcwd_show_start_default = """\
-Changed polling interval to 200ms
+Changed polling interval to 400ms
```

Comment:
why change this value?

Reply:
the corresponding output changed on the test